test/views/ChargeSummarySpec.scala

```diff
@@ -17,26 +17,34 @@
 package views
 
 import assets.FinancialDetailsTestConstants._
-import models.financialDetails.{DocumentDetail, FinancialDetail}
+import models.financialDetails.DocumentDetail
 import org.jsoup.Jsoup
 import org.jsoup.nodes.Document
 import play.twirl.api.Html
 import testUtils.ViewSpec
-import views.html.chargeSummary
-
 import java.time.LocalDate
 
+import models.chargeHistory.ChargeHistoryModel
+import views.html.chargeSummary
+
 class ChargeSummarySpec extends ViewSpec {
 
-	class Setup(documentDetail: DocumentDetail, dueDate: Option[LocalDate] = Some(LocalDate.of(2019, 5, 15))) {
-		val view: Html = chargeSummary(documentDetail, dueDate, mockImplicitDateFormatter, "testBackURL")
+	class Setup(documentDetail: DocumentDetail, dueDate: Option[LocalDate] = Some(LocalDate.of(2019, 5, 15)), paymentEnabled: Boolean = true, chargeHistory: List[ChargeHistoryModel] = List()) {
```

Comment:
paymentEnabled feature has been removed, so we should remove any new additions of paymentEnabled from this class and the tests/it